src/components/SideMenu.jsx

```diff
@@ -91,7 +91,7 @@ export default class SideMenu extends React.Component {
                 >
                   Info
                 </Button>
-                <Button
+                <Button style={{position: "absolute", bottom: "10px"}}
```

Comment:
I generally recommend using styles buttons for this.
Ex. "const BottomButton = styled(Button)" :)